src/stream/src/executor/aggregation/distinct.rs

```diff
@@ -159,6 +159,10 @@ impl<S: StateStore> ColumnDeduplicater<S> {
             }
         }
 
+        // if we determine to flush to the table when processing every chunk instead of barrier
+        // coming, we can evict all including current epoch data.
+        self.cache.evict_except_cur_epoch();
```

Comment:
This can be `evict()`, since `ColumnDeduplicater` will keep the cache synced with state table in chunk granularity.

Reply:
Yes, but I notice there is a "todo" and do not know if we will break it(flush the table in `dedup` method) :thinking: